cfpb/cfgov-refresh
# Remove duplicate OAH social media template

The BAH code includes its own social media template which is mostly a duplicate of the generic one. The BAH one is only used on one specific page, which is [the Explore Rates page](https://www.consumerfinance.gov/owning-a-home/explore-rates/).

This commit removes the custom module in favor of using [the generic one](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/jinja2/v1/_includes/molecules/social-media.html). In practice this makes no difference. The generic one happens to include an (optional) print option, that is not used on this specific page.

## How to test this PR

Visit http://localhost:8000/owning-a-home/explore-rates/ and confirm that it looks the same as [the live page](https://www.consumerfinance.gov/owning-a-home/explore-rates/) and that the share links are correct.

## Notes and todos

This change will make it easier to add proper Spanish support to the social media molecule

## Checklist

  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
